handout/routing/routerlink.md

```diff
@@ -2,7 +2,7 @@
 
 ## RouterLink ##
 
-Links to routes can be added using the `RouterLink` directive, which is part of `ROUTER_DIRECTIVES` along with `RouterOutlet`.
+Add links to routes using the `RouterLink` directive, which is part of `ROUTER_DIRECTIVES` along with `RouterOutlet`.
```

Comment:
Trying to think of how to best explain this. To me, this reads like we are creating new routes, but what the RouterLink does is create a link to an existing route.